### PR TITLE
docs(migration): general improvements

### PR DESCRIPTION
**Pull request by Narretz**

### CODE REVIEW

**Comment by petebacondarwin** [comment]
LGTM. Thanks for reviewing and updating this @Narretz

File: docs/content/guide/migration.ngdoc [hunk 1]
```diff
@@ -138,11 +139,13 @@ class based animations (animations triggered via ngClass) in order to ensure tha
 ## Forms (`ngMessages`, `ngOptions`)
 
 ### ngMessages
-The ngMessages module has also been subject to an internal refactor to allow the feature to be more flexible
+The ngMessages module has also been subject to an internal refactor to allow it to be more flexible
```
Comment (gkalpak) [inline docs/content/guide/migration.ngdoc]: _ngMessages_  -->  _\`ngMessages\`_ ?

File: docs/content/guide/migration.ngdoc [hunk 2]
```diff
@@ -168,11 +171,14 @@ before and after it.
 
 ### ngOptions
 
+The `ngOptions` directive has also been refactored and as result some long-standing bugs
```
Comment (gkalpak) [inline docs/content/guide/migration.ngdoc]: _as result_  -->  _as a result_

File: docs/content/guide/migration.ngdoc [hunk 3]
```diff
@@ -218,9 +222,15 @@ is marked as optional and the attribute is not specified, no function will be ad
 
 Due to [38fbe3ee](https://github.com/angular/angular.js/commit/38fbe3ee8370fc449b82d80df07b5c2ed2cd5fbe),
 `$cookies` will no longer expose properties that represent the current browser cookie
-values.
+values. `$cookies`no longer polls the browser for changes to the cookies and ***no longer copies
```
Comment (gkalpak) [inline docs/content/guide/migration.ngdoc]: _\`$cookies\`no_  -->  _\`$cookies\` no_

File: docs/content/guide/migration.ngdoc [hunk 4]
```diff
@@ -218,9 +222,15 @@ is marked as optional and the attribute is not specified, no function will be ad
...
+cookie values onto the `$cookies` object***.
 
-The new API on `$cookies` includes:
+This was changed because the polling is expensive and caused issues with the `$cookies` properties
+not synchronizing correctly with the actual browser cookie values (The reason the polling
+was originally added was to allow communication between different tabs,
+but there are better ways to do this today, for example `localStorage`).
```
Comment (gkalpak) [inline docs/content/guide/migration.ngdoc]: _)._  -->  _.)_

File: docs/content/guide/migration.ngdoc [hunk 5]
```diff
@@ -229,17 +239,8 @@ The new API on `$cookies` includes:
...
-The polling is expensive and caused issues with the `$cookies` properties not
-synchronizing correctly with the actual browser cookie values.
-
-The reason the polling was originally added was to allow communication between
-different tabs, but there are better ways to do this today (for example `localStorage`).
-
-Now you must explictly use the methods above in order to access cookie data. This also means that
-you can no longer watch the any properties on `$cookies` for changes to detect for any changes
+You must explictly use the methods above in order to access cookie data. This also means that
+you can no longer watch the any properties on `$cookies` to detect any changes
```
Comment (gkalpak) [inline docs/content/guide/migration.ngdoc]: _the any_  -->  _the_ or _any_

**Comment by gkalpak** [comment]
Just pointed out a few typos.

:+1: @Narretz

**Comment by petebacondarwin** [comment]
@Narretz - can you fix the typos and merge this?

**Comment by Narretz** [comment]
@petebacondarwin Sure, I will later today

**Pull request closed without merging**